crates/sui/src/client_commands.rs

```diff
@@ -607,6 +609,54 @@ pub enum SuiClientCommands {
         #[clap(long)]
         address_override: Option<ObjectID>,
     },
+
+    /// Replay a given transaction to view transaction effects. Set environment variable MOVE_VM_STEP=1 to debug.
+    #[clap(name = "replay-transaction")]
+    ReplayTransaction {
+        /// The rpc url for a non-pruning fullnode to use for fetching the transaction dependencies
+        #[arg(long = "rpc")]
+        rpc_url: String,
```

Comment:
The CLI has a notion of an active network, which corresponds to an RPC URL -- can we rely on that as the RPC to connect to?